src/DataCollectors/Microsoft.TestPlatform.Extensions.EventLogCollector/CollectorNameValueConfigurationManager.cs

```diff
@@ -118,8 +118,7 @@ public CollectorNameValueConfigurationManager(XmlElement configurationElement)
                     return null;
                 }
 
-                string settingValue;
-                this.nameValuePairs.TryGetValue(name, out settingValue);
+                this.nameValuePairs.TryGetValue(name, out string settingValue);
```

Comment:
Most of these could be replaced by `out var` please do so where applicable.

```suggestion
                this.nameValuePairs.TryGetValue(name, out var settingValue);
```

Reply:
Resolved where i found some